startup/99-helper_functions.py

```diff
@@ -166,3 +184,19 @@ def wait_for_peaks(pool_interval, timeout, peaks, peaks_fields=None):
                 break
             else:
                 yield from bps.sleep(pool_interval)
+
+def _block_beam(block_beam_bit): ##TODO move this to a child of inout
+    if block_beam_bit: 
+        if inout.status.get() =="Not Inserted":
+            yield from mv(inout, "In")
+    else:
+        if inout.status.get() =="Inserted":
+            yield from mv(inout, "Out")
+
+def block_beam():
```

Comment:
```suggestion
def block_beam():
"""Helper plan to move inout to block beam. Aliased to beam_block()."""
```

Reply:
The alias info might be difficult to keep consistent with future changes.  Consider moving that info to a comment where the aliases are defined.

```suggestion
def block_beam():
"""Helper plan to move inout to block beam."""
```